tests/resources/components/unit_type_variable_schema.json

```diff
@@ -0,0 +1,171 @@
+            },
+            "validFrom": {
+              "$ref": "#/definitions/dateType"
+            },
+            "validUntil": {
+                "oneOf": [
+                    {"type": "null"},
+                    {"$ref": "#/definitions/dateType"}
+                ]
+            }
+          },
+          "required": [
+            "code",
+            "categoryTitle",
+            "validFrom"
+          ]
+        }
+      },
+      "sentinelAndMissingValuesType": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "code": {
+              "type": "string"
+            },
+            "categoryTitle": {
+              "$ref": "#/definitions/multilingualStringType"
+            },
+            "codeType": {
+              "type": "string",
+              "enum": [
+                "MISSING",
+                "SENTINEL"
+              ]
+            }
+          },
+          "required": [
+            "code",
+            "categoryTitle"
+          ]
+        }
+      },
+      "valueDomainType": {
+        "type": "object",
+        "properties": {
+          "description": {
+            "$ref": "#/definitions/multilingualStringType"
+          },
+          "measurementType": {
+            "type": "string",
+            "enum": [
+              "CURRENCY",
+              "WEIGHT",
+              "LENGTH",
+              "HEIGHT",
+              "GEOGRAPHICAL"
+            ]
+          },
+          "measurementUnitDescription": {
+            "$ref": "#/definitions/multilingualStringType"
+          },
+          "uriDefinition": {
+            "type": "array",
+            "items": [
+              {
+                "type": [
+                  "string",
+                  "null"
+                ]
+              }
+            ]
+          },
+          "codeList": {
+            "$ref": "#/definitions/codeListType"
+          },
+          "sentinelAndMissingValues": {
+            "$ref": "#/definitions/sentinelAndMissingValuesType"
+          }
+        },
+        "anyOf": [
+          {
+            "required": [
+              "codeList"
+            ]
+          },
+          {
+            "required": [
+              "description"
+            ]
+          }
+        ]
+      }
+    },
+  
+    "type": "object",
+    "properties": {
+        "shortName": {"type": "string"},
+        "name": {"$ref": "#/definitions/multilingualStringType"},
+        "description": {"$ref": "#/definitions/multilingualStringType"},
+        "dataType": {
+            "type": "string",
+            "enum": ["STRING", "LONG", "DATE", "DOUBLE"]
+        },
+        "format": {"type": "string"},
+        "unitType": {
+            "type": "object",
+            "properties": {
+                "shortName": {"type": "string"},
+                "name": {"$ref": "#/definitions/multilingualStringType"},
+                "pseudonymized": {"type": "boolean"},
```

Comment:
With this name, the code in the pipeline would read `if (pseudonymized) then pseudonymize().` I think this name could indicate that pseudonymization is not yet done, e.g. "requiresPseudonymization".

Reply:
This is sensible 👍🏻 Will change!